pypeclub/OpenPype
# General: Collect loaded versions skips not existing representations

## Brief description
Collector does not crash if there is container with representation id that is not available in mongo.

## Description
First check if representation document was found before creating the item. This is edge case that should never happen, but  there is no reason to crash publishing, especially when artist probably can't do much with that.

## Testing notes:
1. Change manually representation id of any referenced container in the scene (e.g. in maya)
2. Run publishing
3. Instead of crashing should just warn that the representation document was not found in mongo
